<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sharpening</class>
 <widget class="QDialog" name="Sharpening">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>锐化</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>基础算子：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="sliderR">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelR">
     <property name="text">
      <string>半径大小：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="valueR">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>1 </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>均值</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中值</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gaussian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sobel</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Laplacian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LoG</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="3" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
